botpress install command always fail

Hi,

When I try to install a module using the command-line or GUI it fails on GUI I don't get any feedback but command-line says.

for example when I run
`$ botpress install messenger`

I get the following output.

```bash 
 azi  ~/code/test-bot botpress install messenger

[botpress] 	 please wait, we are trying to install your new modules...
[botpress] 	 Installing modules: botpress-messenger
[botpress] 	 ERR npm
[botpress] 	 ERR  WARN
[botpress] 	 ERR  react-codemirror@1.0.0 requires a peer of react@>=15.5 <16 but none was installed.

[botpress] 	 ERR npm 
[botpress] 	 ERR WARN react-codemirror@1.0.0 requires a peer of react-dom@>=15.5 <16 but none was installed.
npm WARN
[botpress] 	 ERR  react-player@0.18.0 requires a peer of react@* but none was installed.
npm WARN react-audio-player@0.5.0 requires a peer of react@15.x but none was installed.

[botpress] 	 ERR npm WARN react-audio-player@0.5.0 requires a peer of react-dom@15.x but none was installed.
npm WARN test-bot@0.0.1 No repository field.
npm WARN test-bot@0.0.1 license should be a valid SPDX license expression

[botpress] 	 ERR 

[botpress] 	 ERR npm
[botpress] 	 ERR  ERR! path /home/azi/code/test-bot/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog/node_modules/are-we-there-yet/node_modules/delegates
npm ERR! code ENOENT

[botpress] 	 ERR npm
[botpress] 	 ERR  ERR! errno -2
npm ERR! syscall
[botpress] 	 ERR  rename

[botpress] 	 ERR npm
[botpress] 	 ERR  ERR! enoent ENOENT: no such file or directory, rename '/home/azi/code/test-bot/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog/node_modules/are-we-there-yet/node_modules/delegates' -> '/home/azi/code/test-bot/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog/node_modules/are-we-there-yet/node_modules/.delegates.DELETE'

[botpress] 	 ERR npm
[botpress] 	 ERR  ERR! enoent This is related to npm not being able to find a file.
npm ERR! 
[botpress] 	 ERR enoent 

[botpress] 	 ERR 

[botpress] 	 ERR npm ERR! A complete log of this run can be found in:
npm ERR!     /home/azi/.npm/_logs/2017-08-07T05_51_18_065Z-debug.log

[botpress] 	 ERR An error occurred during modules installation.
Unhandled rejection undefined
```

@slvnperron here the output for `npm install botpress-messenger --save-dev`

```
 azi  ~/code/test-bot  npm install botpress-messenger --save-dev
npm WARN react-codemirror@1.0.0 requires a peer of react@>=15.5 <16 but none was installed.
npm WARN react-codemirror@1.0.0 requires a peer of react-dom@>=15.5 <16 but none was installed.
npm WARN react-player@0.18.0 requires a peer of react@* but none was installed.
npm WARN react-audio-player@0.5.0 requires a peer of react@15.x but none was installed.
npm WARN react-audio-player@0.5.0 requires a peer of react-dom@15.x but none was installed.
npm WARN test-bot@0.0.1 No repository field.
npm WARN test-bot@0.0.1 license should be a valid SPDX license expression

npm ERR! path /home/azi/code/test-bot/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog/node_modules/are-we-there-yet/node_modules/delegates
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/home/azi/code/test-bot/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog/node_modules/are-we-there-yet/node_modules/delegates' -> '/home/azi/code/test-bot/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog/node_modules/are-we-there-yet/node_modules/.delegates.DELETE'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/azi/.npm/_logs/2017-08-07T09_14_10_320Z-debug.log

```

I have just the same results, can't use botpress at all
1. npm installing botpress messenger failing on the same just like for @azeemhassni (react-codemirror)
2. rm -rf directory, npm cache clear, reinstall with botpress, then as a workaround, sudo npm installing sqlite before any other dependencies also fails, 3.1.8 version can't be found, latest failing with this: 

```
npm ERR! path /Users/.../dev/chatbot-botpress/node_modules/.staging/npmlog-6241e0f4
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/Users/.../dev/chatbot-botpress/node_modules/.staging/npmlog-6241e0f4' -> '/Users/.../dev/chatbot-botpress/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/npmlog'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/.../.npm/_logs/2017-09-04T07_11_42_521Z-debug.log
```

node: 8.4.0
npm: 5.3.0
macOS Sierra 10.12.5

Any idea?

@e-beach yes, `npm install sqlite3@3.1.8` fails with the same error below, so the problem is not with botpress, but because of this dependency i cannot add messenger to botpress, therefore using it (`npm ls` is pretty cool though, thanks for the hint!)

```
> sqlite3@3.1.8 install /Users/.../dev/chatbot-botpress/node_modules/sqlite3
> node-pre-gyp install --fallback-to-build

module.js:491
    throw err;
    ^

Error: Cannot find module 'osenv'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/.../dev/chatbot-botpress/node_modules/sqlite3/node_modules/node-pre-gyp/node_modules/nopt/lib/nopt.js:11:13)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
npm WARN react-codemirror@1.0.0 requires a peer of react@>=15.5 <16 but none was installed.
npm WARN react-codemirror@1.0.0 requires a peer of react-dom@>=15.5 <16 but none was installed.
npm WARN pixelhijack-chatbot-botpress@0.0.1 No repository field.
npm WARN pixelhijack-chatbot-botpress@0.0.1 license should be a valid SPDX license expression

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sqlite3@3.1.8 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sqlite3@3.1.8 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "socket.io-client": "^2.0.3",
     "socketio-jwt": "^4.5.0",
     "source-map-support": "^0.4.6",
-    "sqlite3": "^3.1.8",
+    "sqlite3": "3.1.8",
     "universal-analytics": "^0.4.8",
     "uuid": "^3.0.0",
     "winston": "^2.2.0"

